packages/graph/hash_graph/lib/graph/src/ontology/types/link_type/mod.rs

```diff
@@ -67,7 +66,7 @@ mod tests {
     fn test_link_type_schema(schema: &serde_json::Value) -> LinkType {
         let link_type: LinkType = serde_json::from_value(schema.clone()).expect("invalid schema");
         assert_eq!(
-            serde_json::to_value(link_type.clone()).expect("could not serialize"),
+            serde_json::to_value(link_type.clone()).expect("Could not serialize"),
```

Comment:
Error messages should not be capitalized.
```suggestion
            serde_json::to_value(link_type.clone()).expect("could not serialize"),
```

Could you also please look at other locations you changed?

Reply:
We're discussing this offline and struggling to find guidelines or conventions for panics vs errors. Likely going to go with lower case

Reply:
Have found and replaced all in the project, if there are some expects with the string on a newline I might have missed them